tracking/include/pcl/tracking/impl/pyramidal_klt.hpp

```diff
@@ -543,7 +543,8 @@ pcl::tracking::PyramidalKLTTracker<PointInT, IntensityT>::track (const PointClou
       det = 1.f/det;
       next_pt -= half_win;
 
-      Eigen::Array2f prev_delta;
+      Eigen::Array2f prev_delta (0, 0);
+      Eigen::Array2i inext_pt (0, 0);
```

Comment:
Would it not make more sense to move `inext_pt` inside the for loop. It's always being written at the beginning of the loop anyway.

Reply:
After all, it actually is used outside the loop, line 614.

Reply:
I think that's actually a bug in line 614. It should be `inext_point` which is initialized in line 592. At least from looking at the code that's the impression that it gives. Otherwise lines 609-610 are useless.